workspaces/resource-optimization/plugins/resource-optimization-backend/README.md

```diff
@@ -1,14 +1,17 @@
-# resource-optimization
+# Resource Optimization back-end plugin
 
 Welcome to the resource-optimization backend plugin!
 
 _This plugin was created through the Backstage CLI_
 
 ## Getting started
 
-Your plugin has been added to the example app in this repository, meaning you'll be able to access it by running `yarn
-start` in the root directory, and then navigating to [/resourceOptimizationPlugin/health](http://localhost:7007/api/resourceOptimizationPlugin/health).
+This workspace includes a [devcontainer](../../.devcontainer/devcontainer.json) configuration, this is the recommended way for setting up a local dev-environment. Just open the [resource-optimization.code-workspace](../../resource-optimization.code-workspace) file with VSCode and choose to open this workspace in the preconfigured devcontiner when teh prompt appears.
 
-You can also serve the plugin in isolation by running `yarn start` in the plugin directory.
+The plugin has been added to the example app in this workspace, meaning you'll be able to access it by running `yarn
```

Comment:
@jkilzi can you please share what do you mean by `example app` here?

Reply:
This is how they refer to the `package/app`. It is a term commonly used in other plugins as well.